code/modules/hydroponics/grown/peach.dm

```diff
@@ -0,0 +1,91 @@
+//Peaches
+/obj/item/seeds/peach
+	name = "pack of peach seeds"
+	desc = "Contains seeds that grow into a lovely peach tree."
+	icon_state = "seed-peach"
+	species = "peach"
+	plantname = "Peach Tree"
+	product = /obj/item/food/grown/peach
+	lifespan = 20
+	endurance = 25
+	yield = 5
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_grow = "peach-grow"
+	icon_dead = "peach-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	mutatelist = list(/obj/item/seeds/peach/impeach)
+	reagents_add = list(/datum/reagent/consumable/nutriment/vitamin = 0.04, /datum/reagent/consumable/nutriment = 0.1, /datum/reagent/water = 0.08)
+
+/obj/item/food/grown/peach
+	seed = /obj/item/seeds/peach
+	name = "peach"
+	desc = "Way better than canned peaches."
+	icon_state = "peach"
+	foodtypes = FRUIT
+	juice_results = list(/datum/reagent/consumable/peachjuice = 0)
+	tastes = list ("peaches" = 1)
+	wine_power = 35
+
+//impeachs
+/obj/item/seeds/peach/impeach
+	name = "pack of impeach seeds"
+	desc = "A pack of impeach seeds."
+	icon_state = "seed-impeach"
+	species = "peach"
+	plantname = "Impeach Tree"
+	product = /obj/item/food/grown/peach/impeach
+	icon_harvest = "impeach-harvest"
+	mutatelist = list()
+	rarity = 5
+
+/obj/item/food/grown/peach/impeach
+	seed = /obj/item/seeds/peach/impeach
+	name = "impeach"
+	desc = "A quick way to impeach your pets, friends or even your whole family!"
+	icon_state = "impeach"
+	var/activated = FALSE //Has the impeach been activated yet?
```

Comment:
You have generated new variables/procs/defines without proper documentation for the autodoc to work.

Please click [here ](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)for a guide on how to properly document these items.